packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -57,12 +57,20 @@ function printAttributes(attributes, print, indent, colors, opts) {
 }
 
 const print = (
-  element: any,
+  element: HTMLElement | Text | Comment,
   print: Print,
   indent: Indent,
   opts: Options,
   colors: Colors,
-) => {
+): string => {
+  if (element instanceof Text) {
```

Comment:
Yes, it works in the browser (and "wannabe-browsers", like jsdom), but not in Node. The issue is if you want to use the plugin from node without polluting global namespace